app/views/idv/sessions/new.html.slim

```diff
@@ -1,11 +1,11 @@
-- title @view_model.title
+- title t('idv.titles.sessions')
 
-h1.h3 = @view_model.title
+h1.h3 = t('idv.titles.sessions')
 
 p = link_to t('links.access_help'),
   'https://login.gov/help/privacy-and-security/how-does-logingov-protect-my-data/'
 
-= simple_form_for(@view_model.idv_form, url: idv_session_path,
+= simple_form_for(@idv_form, url: idv_session_path,
```

Comment:
One of the patterns I'm playing with in the 2fa modularization effort is making the `form` a property of the presenter object. This is helpful especially when parts of the presenter might depend on information in the form, such as error states. But in that case, I'd make things like the `dob` (line 22 below) a property of the presenter, and have the presenter figure out how to get it from the form.

Reply:
Since we're focusing on adding fail states to the IdV flow here, I'd call refactoring the form / presenter object relationship for this kind of out of scope. We can come back and investigate though.